src/data/news/coverage.yml

```diff
@@ -1,3 +1,19 @@
+-
+  Title: "Decred: one of the oldest cryptos convinces with principles (German)"
```

Comment:
This looks a bit weird without title-case
![Screenshot from 2023-11-07 08-45-24](https://github.com/decred/dcrweb/assets/6762864/5e2b1902-67ba-42ca-8351-3ad4c4428a53)

Reply:
Thanks for testing how it looks. Changed to title case.